examples/release_highlights/plot_release_highlights_1_0.py

```diff
@@ -0,0 +1,184 @@
+#
+# .. figure:: /auto_examples/linear_model/images/sphx_glr_plot_quantile_regression_002.png
+#    :target: ../auto_examples/linear_model/plot_quantile_regression.html
+#    :align: center
+#    :scale: 50%
+
+##############################################################################
+# A more flexible plotting API
+# --------------------------------------------------------------------------
+# :class:`metrics.ConfusionMatrixDisplay`,
+# :class:`metrics.PrecisionRecallDisplay`, and :class:`metrics.DetCurveDisplay`
+# now expose two class methods: `from_estimator` and `from_predictions` which
+# allow users to create a plot given the predictions or an estimator. This
+# means the corresponsing `plot_*` functions are deprecated. Please check
+# :ref:`example one
+# <sphx_glr_auto_examples_model_selection_plot_confusion_matrix.py>` and
+# :ref:`example two
+# <sphx_glr_auto_examples_classification_plot_digits_classification.py>` for
+# examples of how to use the new plotting functionalities.
+
+##############################################################################
+# Online One-Class SVM
+# --------------------------------------------------------------------------
+# The new class :class:`sklearn.linear_model.SGDOneClassSVM` implements an online
+# linear version of the One-Class SVM using a stochastic gradient descent.
+# Combined with kernel approximation techniques,
+# :class:`sklearn.linear_model.SGDOneClassSVM` can be used to approximate the
+# solution of a kernelized One-Class SVM, implemented in
+# :class:`sklearn.svm.OneClassSVM`, with a linear complexity in the number of
+# samples. Note that the complexity of a kernelized One-Class SVM is at best
+# quadratic in the number of samples.
+# :class:`sklearn.linear_model.SGDOneClassSVM` is thus well suited for datasets
+# with a large number of training samples (> 10,000) for which the SGD variant
+# can be several orders of magnitude faster. Please check this :ref:`example
+# <sphx_glr_auto_examples_miscellaneous_plot_anomaly_comparison.py` to see how
+# it's used, and :ref:`User Guide <sgd_online_one_class_svm>` for more details.
+
+# .. figure:: /auto_examples/miscellaneous/images/sphx_glr_plot_anomaly_comparison_001.png
+#    :target: ../auto_examples/miscellaneous/plot_anomaly_comparison.html
+#    :align: center
+
+##############################################################################
+# Histogram-based Gradient Boosting Models are now stable
+# --------------------------------------------------------------------------
+# :class:`~sklearn.ensemble.HistGradientBoostingRegressor` and
+# `ensemble.HistGradientBoostingClassifier` are no longer experimental and can
+# simply be imported and used as::
+#
+#     from sklearn.ensemble import HistGradientBoostingClassifier
+
+##############################################################################
+# New documentation improvements
+# ------------------------------
+# TODO
```

Comment:
@lucyleeow @cmarmo is there anything you'd like to be included here?

Reply:
Sorry, I haven't kept up to date with changes in this release (woo 1.0!)

Reply:
Thanks @adrinjalali for checking. I'm not up to date either... was I ever?... ;)